package.json

```diff
@@ -5,13 +5,14 @@
   "main": "index.js",
   "dependencies": {
     "@hyperswarm/hypersign": "^2.0.0",
-    "dht-rpc": "^4.2.0",
+    "dht-rpc": "git+https://github.com/davidmarkclements/dht-rpc.git#turn-non-ephemeral",
```

Comment:
TODO: update to next dht-rpc release when https://github.com/mafintosh/dht-rpc/pull/9 lands